src/components/textInput.re

```diff
@@ -9,6 +9,141 @@ external view : ReasonReact.reactClass = "TextInput";
 
 [@bs.send] external blur : ReasonReact.reactRef => unit = "";
 
+module AutoCapitalize = {
+  type t = [
+    | `none
+    | `sentences
+    | `words
+    | `characters
+  ];
+let to_string = x => {
+        switch (x) {
```

Comment:
You could use bs.deriving jsconverter instead of writing to_string methods everywhere https://bucklescript.github.io/docs/en/generate-converters-accessors